README.md

```diff
@@ -1,12 +1,51 @@
+## About this package
```

Comment:
This section feels weird/wrong to have in a general readme vs. the NuGet package specific readme.